nuxeo-core/nuxeo-core-test/src/main/java/org/nuxeo/ecm/core/test/StorageConfiguration.java

```diff
@@ -177,6 +183,17 @@ protected void clearMongoDB(MongoDBRepositoryDescriptor descriptor) throws Unkno
         }
     }
 
+    protected void initExternal() {
+        // Get DBSHelper by reflection
+        String className = String.format("org.nuxeo.ecm.storage.%s.DBSHelperImpl", coreType);
```

Comment:
We changed that when we extracted marklogic from Nuxeo repository, also nuxeo-core-storage-marklogic became nuxeo-storage-marklogic, does we get back on this ? (and all other changes to add core at right position)